fitnesse/resources/bootstrap-plus/js/bootstrap-plus.js

```diff
@@ -216,5 +222,109 @@ $( document ).ready(function() {
                 $('#autoSave-switch').addClass('fa-toggle-on');
             }
         }
+    // Distinction made between different types of pages so it doesn't apply to all parents
+    // Add hover function to suite page to show add tag button
+    $('.suite').parent().hover(
+        function() {
+            $(this).find('.addTag:first').css("visibility", "visible");
+        }, function() {
+            $(this).find('.addTag:first').css("visibility", "hidden");
+        }
+    );
+    // Add hover function to static page to show add tag button
+    $('.static').parent().hover(
+        function() {
+            $(this).find('.addTag:first').css("visibility", "visible");
+        }, function() {
+            $(this).find('.addTag:first').css("visibility", "hidden");
+        }
+    );
+    // Add hover function to test page to show add tag button
+    $('.test').parent().hover(
+        function() {
+            $(this).find('.addTag:first').css("visibility", "visible");
+        }, function() {
+            $(this).find('.addTag:first').css("visibility", "hidden");
+        }
+    );
+
+    // Click add tag function
+    $('.addTag').click(function() {
+        addTagInput($(this));
+    });
 });
 
+function addTagInput(currentAddTagButton){
+    //Remove all existing tag input fields
+    $('.tagInputOverview').remove();
+
+    //Add input field
+    $(currentAddTagButton).after('<input type="text" class="tagInputOverview">');
+
+    //Add focus after clicking button
+    $('.tagInputOverview').focus();
+
+    $('.tagInputOverview').keyup(function(event) {
+        //If "Enter" button is pressed
+        if (event.keyCode == 13) {
+            //Get current input value & replace empty spaces at the end of input
+            const inputValue = $('.tagInputOverview').val().replace(/\s+\S*$/, "");
```

Comment:
Ik denk dat `$('.tagInputOverview').val().trim()` sneller is dan een regex :-)

Reply:
Regex aangepast naar de Trim functie.